packages/lesswrong/lib/collections/manifoldProbabilitiesCaches/collection.ts

```diff
@@ -0,0 +1,20 @@
+import schema from './schema';
+import { createCollection } from "../../vulcan-lib";
+import { addUniversalFields } from '../../collectionUtils';
+import { ensureCustomPgIndex, ensureIndex } from '../../collectionIndexUtils';
+
+export const ManifoldProbabilitiesCaches = createCollection({
+  collectionName: 'ManifoldProbabilitiesCaches',
+  typeName: 'ManifoldProbabilitiesCache',
+  schema,
+  logChanges: false,
+  writeAheadLogged: false,
+});
+
+addUniversalFields({collection: ManifoldProbabilitiesCaches});
+
+void ensureIndex(ManifoldProbabilitiesCaches, {
```

Comment:
I think this doesn't need `void`, since `ensureIndex` is not itself an `async` function